egs/chime6/asr1/local/run_gss.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright 2015, Mitsubishi Electric Research Laboratories, MERL (Author: Shinji Watanabe)
+
+. ./cmd.sh
+if [ -f ./path.sh ]; then . ./path.sh; fi
+
+# Config:
+cmd=run.pl
+nj=4
+multiarray=outer_array_mics   #-- 16 channels for gss; usually better than 24 channels gss
+#multiarray=True              #-- 24 channels for gss
+bss_iterations=5
```

Comment:
24 channel GSS seems to be better from your paper when also used with train GSS option. This parameter "multiarray" doesn't seem to be passed from "run.sh", can you please add options in "run.sh" and pass it from there.

Reply:
Thanks for your suggestion!
Indeed, according to our results, WER on dev_gss24 is better than on dev_gss12. However, this gss24 is built according to the STC pipeline (as I mentioned above) that is different from the baseline gss24. In our local experiments, the baseline gss24 performed about 0.5-0.6% worse than the gss12. Yet, I can add an option with proper comments for it in run.sh